scripts/setup_irods.py

```diff
@@ -263,7 +265,20 @@ def setup_service_account(irods_config, irods_user, irods_group):
     #owner of top-level directory changed, clear the cache
     irods_config.clear_cache()
 
-def setup_server_config(irods_config):
+def setup_run_delay_server_configuration_option(irods_config, run_delay_server=None):
+    l = logging.getLogger(__name__)
+    l.debug('Setting up advanced_settings.run_delay_server configuration option ...')
+
+    # Determine if the rule execution (delay) server should be launched on startup.
+    if run_delay_server != None:
+        irods_config.server_config['advanced_settings']['run_delay_server'] = run_delay_server.lower() in ['true', 'y', 'yes', '1']
```

Comment:
parentheses needed/helpful around rhs of this assignment?   same as two lines down?

Reply:
done